.github/workflows/orbit-retryable-monitor.yml

```diff
@@ -0,0 +1,60 @@
+name: Orbit Retryable Tracker
+
+on:
+  schedule:
+    # Run everyday at 12:00 PM UTC
+    - cron: '0 12 * * *'
+
+env:
+  NEXT_PUBLIC_INFURA_KEY: ${{ secrets.NEXT_PUBLIC_INFURA_KEY }}
+  SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
+  SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL }}
+
+jobs:
+  run-alerting:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Bridge
+      uses: actions/checkout@v3
+      with:
+        repository: OffchainLabs/arbitrum-token-bridge
+        ref: 'feat-enable-orbit-retryable-monitoring' # the branch where our alerting code is present
```

Comment:
Before merging, it'd be great if we ship this branch as well, but not a hard requirement as specifying the custom branch in actions works too.